include/itkRegionCompetitionImageFilter.hxx

```diff
@@ -445,7 +445,7 @@ RegionCompetitionImageFilter<TInputImage,TOutputImage>
 
   NeigborOffsetIterator neighborItr = this->m_NeighborBufferOffset.begin();
 
-  const OutputImagePixelType backgroundValue = 0;  // FIXME: replace with NO-Label.
+  constexpr OutputImagePixelType backgroundValue  = 0;  // FIXME: replace with NO-Label.
```

Comment:
extra space before the equals sign

Reply:
Thanks @dzenanz .  I'm not going to fix those spaces now.  Perhaps in future code cleanups.